users/views.py

```diff
@@ -4,6 +4,23 @@
 from rest_framework import permissions
 from users.permissions import IsAdminUser, IsOwnerOrAdmin
 
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.reverse import reverse
+
+
+@api_view(['GET'])
+def api_root(request, format=None):
```

Comment:
Why api_root?

Reply:
This api_root is a custom version of default api_root supplied by Django REST. I tried to customize 'main page' for different kind of user.